docs/collect/system-metrics.md

```diff
@@ -0,0 +1,62 @@
+<!--
+title: "Collect system metrics with Netdata"
+sidebar_label: "System metrics"
+description: "Netdata collects thousands of metrics from physical and virtual systems, IoT/edge devices, and containers with zero configuration."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/collect/system-metrics.md
+-->
+
+# Collect system metrics with Netdata
+
+Netdata collects thousands of metrics directly the operating systems of physical and virtual systems, IoT/edge devices,
+and [containers](/docs/collect/container-metrics.md) with zero configuration.
+
+To gather system metrics, Netdata uses roughly a dozen plugins, each of which has one or more collectors for very
+specific metrics exposed by the host. Most metrics Netdata users interface with most for health monitoring and
```

Comment:
Suggest changing "users interface" to "users interact". I stumbled on this, reading it as "user interface".